src/libraries/Microsoft.PowerFx.Connectors/Environment/CdpTabularService.cs

```diff
@@ -16,19 +16,27 @@ namespace Microsoft.PowerFx.Connectors
     // Implements CDP protocol for Tabular connectors
     public class CdpTabularService : TabularService
     {
-        public string DataSetName { get; }
+        public string Name { get; private set; }
```

Comment:
>Name [](http://example.com/codeflow?start=22&length=4)

Name is too ambigious... 

Can we comment difference between these names (and maybe with a common example of each)

Reply:
Will do in future PR